api.py

```diff
@@ -94,6 +112,22 @@ def fetch_task_gql(mentor_id: str, question_id) -> GQLQueryBody:
         },
     }
 
+def convert_upload_task_gql(upload_task):
```

Comment:
this probably isn't required, it's better to work with the map everywhere and avoid this map<->list conversion

Reply:
[42855af](https://github.com/mentorpal/mentor-upload-processor/pull/2/commits/42855af608a5db04702f760347c173f9f29059f6)